projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"io/fs"
+	"os"
+)
+
+func Execute()(error) {
+	var fileInfo fs.FileInfo
+	var err error
```

Comment:
It's unusual to pre-declare these variables at the function scope, rather than to declare them where they're first used. Is there a reason you've don this here?

Reply:
Hmm must have been an oversight, there was no reason for that.